keps/sig-auth/0014-dynamic-audit-configuration.md

```diff
@@ -17,8 +17,8 @@ approvers:
   - "@yliaog"
 editor: TBD
 creation-date: 2018-05-18
-last-updated: 2018-07-13
-status: provisional
+last-updated: 2018-07-30
+status: implementable
```

Comment:
Please either revert this or close https://github.com/kubernetes/community/pull/2417

Reply:
closed